buda/entities/subscription_created.proto

```diff
@@ -4,14 +4,20 @@ package buda.entities;
 
 import "buda/entities/uuid.proto";
 import "google/protobuf/timestamp.proto";
-import "buda/entities/subscription.proto";
 import "buda/entities/payment.proto";
 
 message SubscriptionCreated {
     Uuid id = 1;
     Uuid user_id = 2;
-    google.protobuf.Timestamp recorded_at = 3;
-    Subscription subscription = 4;
-    Payment payment = 5;
-
+    Uuid plan_id = 3;
```

Comment:
Hmm... maybe we can chat on this one 😄 

I feels like maybe that SubscriptionCreated would be the only place that the additional nested Subscription message would apply.  If that is the case (and I'm not overlooking something 😅 ), wouldn't it perhaps be cleaner to not have the messages nested?